### PR TITLE
[Monitor] Support the collection and storage of monitoring data.

### LINKED ISSUE #6262: Support the collection and storage of monitoring data
**Describe the solution you'd like**

The master of fe collects the monitoring data of fe and be periodically and stores it in bdbje.

* For time series monitoring data can be obtained by http get，
url:  http://fe_host:http_port/rest/v2/monitor/timeserial/{MonitorType}?start=startTimestamp&end=endTimestamp
the value of 'MonitorType' is {@link Monitor.MonitorType}, the 'start' and 'end' parameters are timestamps that specify the start and end times of the time series data to be retrieved.

The request body is in json format, as follows：
```
{
  "nodes":[
    "host1:http_port", 
    "host2:http_port"
  ],
  "point_num":num
}
```
‘nodes' specifies which nodes to get the monitoring data, the default is all nodes; 'point_num' specifies the number of returned data, the default is 100.

The return result is in json format, as follows:
```
{
    "x_value":[
        "timestamp",
        "timestamp"
    ],
    "y_value":{
        "fe_host1:http_port":[
            "value",
            "value"
        ]
    }
}
```

* For value type monitoring data can be obtained by http get，
url: http://fe_host:http_port/rest/v2/monitor/cluster_info/{ClusterInfoType}
the value of 'ClusterInfoType' is {@link ClusterInfo.ClusterInfoType}

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by luozenglin**
Collect the metrics of each node and store it in bdbje on FE. Then we can get the monitoring data through the http server.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6262) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/test/java/org/apache/doris/metric/collector/MonitorTest.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
+package org.apache.doris.metric.collector;
```
Comment (acelyc111) [inline fe/fe-core/src/test/java/org/apache/doris/metric/collector/MonitorTest.java:1]: License header missed.

**Comment by morningman** [comment]
Please add the performance test of bdbje when saving metric data, and the estimate of data volume that will be stored in bdbje.
And further more, the cleaning and aggregation of historical data has not been solved yet, you can record a TODO

**Comment by caiconghui** [comment]
why not use prometheus to collect monitor data? what's the motivation or benefit to store monitor data in bdbje?

**Comment by morningman** [comment]
> why not use prometheus to collect monitor data? what's the motivation or benefit to store monitor data in bdbje?

Many users did not deploy a monitoring system in the early stage of using Doris, which makes it difficult to troubleshoot some problems. So this function stores a small number of key metrics in a built-in way to help troubleshoot problems.
This is not a substitute feature for the standard monitoring system, but in some cases it helps to locate the problem as soon as possible.

File: fe/fe-core/src/test/java/org/apache/doris/metric/collector/MonitorTest.java (hunk 1 above)
Comment (luozenglin) [reply fe/fe-core/src/test/java/org/apache/doris/metric/collector/MonitorTest.java:18]: done. thanks

**Comment by luozenglin** [comment]
> Please add the performance test of bdbje when saving metric data, and the estimate of data volume that will be stored in bdbje.
> And further more, the cleaning and aggregation of historical data has not been solved yet, you can record a TODO

done

**Review by morningman — APPROVED** [review]
LGTM

**Review by morningman — CHANGES_REQUESTED** [review]
Still has problem

**Comment by morningman** [comment]
abondoned

**Pull request closed without merging**